src/GovCmsCkanDatasetParser.inc

```diff
@@ -330,6 +331,32 @@ class GovCmsCkanDatasetParser {
   }
 
   /**
+   * A helper to cleanse a value by removing new lines.
+   *
+   * @param string $value
+   *   The value to cleanse.
+   *
+   * @return string
+   *   The cleansed value.
+   */
+  public function cleanseValue($value = NULL) {
+    return str_replace(array("\n", "\r"), ' ', $value);
```

Comment:
Technically what is there would have still worked, we would have then just had 2 spaces :) updated now though.